fvm/runtime/reusable_cadence_runtime.go

```diff
@@ -179,13 +180,17 @@ type ReusableCadenceRuntimePool struct {
 func newReusableCadenceRuntimePool(
 	poolSize int,
 	config runtime.Config,
+	chainID flow.ChainID,
 	newCustomRuntime func() runtime.Runtime,
 ) ReusableCadenceRuntimePool {
 	var pool chan *ReusableCadenceRuntime
 	if poolSize > 0 {
 		pool = make(chan *ReusableCadenceRuntime, poolSize)
 	}
 
+	// Enable account linking on all networks except Mainnet
+	config.AccountLinkingEnabled = chainID != flow.Mainnet
```

Comment:
minor: I would prefer this set being higher up so chainID doesn't need to get sent so deep just for this. 
But since this is just temporary for 0.29 its ok.

Reply:
+1 if you're porting this to head

Reply:
I wasn't sure how to enforce this setting consistently everywhere. It would be great if someone from the execution team with more knowledge about flow-go/FVM could advise / perform the refactor.